packages/ui/src/ui/containers/SettingsPanel/settings-description.tsx

```diff
@@ -457,6 +458,32 @@ function useSettings(cluster: string, isAdmin: boolean): Array<SettingsPage> {
                 ),
             ]),
         ),
+        isAdmin &&
+            makePage(
```

Comment:
If we don't want to allow edit the settings for regular users it is better to put them under `Development` section of settings.